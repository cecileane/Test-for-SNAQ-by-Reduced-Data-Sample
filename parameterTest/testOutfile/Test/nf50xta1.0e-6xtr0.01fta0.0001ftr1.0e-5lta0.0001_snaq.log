optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=0.0001,
                       xtolAbs=1.0e-6, xtolRel=0.01.
 max number of failed proposals = 50, liktolAbs = 0.0001.
rootname for files: nf50xta1.0e-6xtr0.01fta0.0001ftr1.0e-5lta0.0001_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 10701
seed: 10701 for run 1
Sat Oct  1 17:16:05 2016

 BEGIN SNaQ for run 1, seed 10701 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 118 (failures: 50) with -loglik=126.74251 and ht_min=[0.2856,10.0,0.51916,1.51213,0.48942]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 44		 44		 1	 |	 0		 0		 --
mvtarget	 30		 30		 2	 |	 0		 0		 --
chdir		 13		 13		 1	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 30		 22		 2	 |	 2		 2		 --
Total		 118		 110		 7	 |	 2		 2		 2
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 1, -loglik of best 126.74250981978548
((5,(6,#H7:::0.2856030924666545):10.0):0.5191610490423707,(4,(2,(1)#H7:::0.7143969075333455):1.5121344006887287):0.48941914702563993,3);
---------------------
seed: 50530 for run 2
Sat Oct  1 17:16:14 2016

 BEGIN SNaQ for run 2, seed 50530 and hmax 1
Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 83 (failures: 50) with -loglik=126.74251 and ht_min=[0.2856,0.48942,0.51916,10.0,1.51213]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 2		 2		 1	 |	 NA 		 NA 		 NA 
mvorigin	 27		 27		 0	 |	 0		 0		 --
mvtarget	 16		 16		 0	 |	 0		 0		 --
chdir		 7		 7		 2	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 31		 27		 0	 |	 2		 2		 --
Total		 83		 79		 3	 |	 2		 2		 2
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 2, -loglik of best 126.7425098197518
(3,((2,(1)#H7:::0.7143970005107078):1.512133314272998,4):0.48941881535644666,(5,(6,#H7:::0.28560299948929224):10.0):0.5191607563622981);
---------------------

Sat Oct  1 17:16:20 2016
MaxNet is (3,((2,(1)#H7:::0.7143970005107078):1.512133314272998,4):0.48941881535644666,(5,(6,#H7:::0.28560299948929224):10.0):0.5191607563622981); 
with -loglik 126.7425098197518
