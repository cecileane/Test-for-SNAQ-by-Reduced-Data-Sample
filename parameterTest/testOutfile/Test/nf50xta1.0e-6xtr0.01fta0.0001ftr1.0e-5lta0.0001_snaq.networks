(3,((2,(1)#H7:::0.7143970005107078):1.512133314272998,4):0.48941881535644666,(5,(6,#H7:::0.28560299948929224):10.0):0.5191607563622981);, with -loglik 126.7425098197518 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(3,((2,(1,#H7:::0.30999495991009096):2.8161901742583284):0.6426878810113309,4):0.4386277241738692,(5,(6)#H7:::0.690005040089909):1.6943827609378748);, with -loglik 128.5563282757997
(3,(4)#H7:::0.6028298223236244,(5,(6,(1,(#H7:::0.39717017767637564,2):2.458851217723302):1.1551615593697668):0.2214962369968917):0.0270414487615651);, with -loglik 172.65908429827414
(3,(#H7:::0.413654066997004,4):2.0312936404487805,(5,(6,(1,(2)#H7:::0.586345933002996):3.337846765064303):0.153248663206515):0.028112821197565446);, with -loglik 192.5125092696025
((3)#H7:::0.5979257091283167,(2,(1,((5,#H7:::0.40207429087168334):9.633675566663289,6):0.9921833813960088):0.09632069274111753):0.07234639896542153,4);, with -loglik 215.12563479005019
(3,((2,(1,((5)#H7:::0.5415138636658151,6):2.5237977253616064):0.04532057169029576):0.17338315415789265,4):1.6960156012992809,#H7:::0.4584861363341849);, with -loglik 235.46287428814574
