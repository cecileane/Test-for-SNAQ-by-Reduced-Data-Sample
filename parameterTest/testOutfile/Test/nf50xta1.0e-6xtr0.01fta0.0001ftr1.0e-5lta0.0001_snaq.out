(3,((2,(1)#H7:::0.7143970005107078):1.512133314272998,4):0.48941881535644666,(5,(6,#H7:::0.28560299948929224):10.0):0.5191607563622981); -Ploglik = 126.7425098197518
 Dendroscope: (3,((2,(1)#H7):1.512133314272998,4):0.48941881535644666,(5,(6,#H7):10.0):0.5191607563622981);
 Elapsed time: 14.870705358 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 (3,((2,(1)#H7:::0.7143970005107078):1.512133314272998,4):0.48941881535644666,(5,(6,#H7:::0.28560299948929224):10.0):0.5191607563622981);, with -loglik 126.7425098197518
 ((5,(6,#H7:::0.2856030924666545):10.0):0.5191610490423707,(4,(2,(1)#H7:::0.7143969075333455):1.5121344006887287):0.48941914702563993,3);, with -loglik 126.74250981978548
-------
